src/components/Slider/SliderCommons.js

```diff
@@ -0,0 +1,10 @@
+import { createBemBlockHelper } from "../../helpers/bem-helper";
+
```

Comment:
In other files we use the following convention for constants files - ComponentNameConstants.js
Let's rename the file name to be according to this convention.

Reply:
Done!